Portfolio/index.css

```diff
@@ -1 +1,4 @@
-/* Write your code below for PORTFOLIO */
+console.log('Hello World');

```

Comment:
Unexpected text 'body' on line 2. Did you intend to use this as a CSS selector within style tags in HTML or within a CSS file?